.idea/ivy.iml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
-    <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Remote Python 3.8.10 Docker (unifyai/ivy:latest)" jdkType="Python SDK" />
+    <content url="file://$MODULE_DIR$">
```

Comment:
The contents of .idea folder should not be modified. Kindly remove these changes from the commit :)